app/pages/_app.tsx

```diff
@@ -178,34 +196,20 @@ export default function CrosshareApp({
           key="theme"
           dangerouslySetInnerHTML={{
             __html: `
-html, body.light-mode {
-  ${colorThemeString({
-    primary: PRIMARY,
-    link: LINK,
-    errorColor: ERROR_COLOR,
-    verifiedColor: VERIFIED_COLOR,
-    darkMode: false,
-    preservePrimary: false,
-  })}}
+html, body.light-mode, body.dark-mode .reverse-theme {
+  ${lightTheme}
+}
+.reverse-theme, body.dark-mode, body.light-mode .reverse-theme {
+  ${darkTheme}
+}
 @media (prefers-color-scheme: dark) {
```

Comment:
I changed the order of these because media queries do not add css specificity. If the old order was doing something that I'm not realizing, let me know! I think it might be possible to fix this with specificity too.